modules/living_spaces_intranet/living_spaces_intranet.module

```diff
@@ -255,3 +255,21 @@ function living_spaces_intranet_employee_form_submit($form, FormStateInterface $
     ]));
   }
 }
+
+/**
+ * Implements hook_form_BASE_FORM_ID_alter().
+ */
+function living_spaces_intranet_form_user_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+
+  if (isset($form['account']['roles']['#options']) && \Drupal::hasService('delegatable_roles')) {
```

Comment:
you should add the module as a dependency to the module and remove `\Drupal::hasService()` validation.